src/store/modules/auth.js

```diff
@@ -97,6 +104,9 @@ const mutations = {
   updateUserStatus(state, status) {
     state.user.status = status;
   },
+  setAnalyticsAccessToken(state, accessToken) {
```

Comment:
don't we need an unset method that we might need when we have to refresh the token?